Exit code is zero when there are problems with parameters

When there is a parameter error the exit code is 0 (success). The standard exit code for parameter problems is 2. When bioformats2raw is run in an automated fashion it would be very useful to have a non-zero exit code for parameter issues, otherwise it looks like everything succeeded in these cases. Here is an example:

```
$ bioformats2raw --compression-properties "" input.tif output.zarr
Value for option option '--compression-properties' (<String=Object>) should be in KEY=VALUE format but was
Usage: <main class> [-p] [--help] [--keep-memo-files] [--no-hcs] [--[no-]
                    minmax] [--[no-]nested] [--no-ome-meta-export]
                    [--no-original-metadata] [--no-root-group] [--overwrite]
                    [--use-existing-resolutions] [--version] [--debug
$ echo $?
0
```

In some cases, an exception is thrown for bad parameters, e.g.:

```
$ bioformats2raw
Exception in thread "main" picocli.CommandLine$ExecutionException: Error while calling command (com.glencoesoftware.bioformats2raw.Converter@28975c28): java.lang.IllegalArgumentException: Input path not specified
	at picocli.CommandLine.executeUserObject(CommandLine.java:2050)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
	at picocli.CommandLine$AbstractParseResultHandler.handleParseResult(CommandLine.java:2264)
	at picocli.CommandLine.parseWithHandlers(CommandLine.java:2664)
	at picocli.CommandLine.parseWithHandler(CommandLine.java:2599)
	at picocli.CommandLine.call(CommandLine.java:2875)
	at com.glencoesoftware.bioformats2raw.Converter.main(Converter.java:2998)
Caused by: java.lang.IllegalArgumentException: Input path not specified
	at com.glencoesoftware.bioformats2raw.Converter.call(Converter.java:1160)
	at com.glencoesoftware.bioformats2raw.Converter.call(Converter.java:104)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2041)
	... 9 more
$ echo $?
1
```

In this case the exit code is 1 because the program crashed. It would be nice if these cases were handled consistently along with other parameter problems. The `IllegalArgumentException` is meant for arguments to methods, not for arguments to a CLI. I suggest printing the error, showing the usage information, and returning exit code 2.

--- a/src/main/java/com/glencoesoftware/bioformats2raw/Converter.java
+++ b/src/main/java/com/glencoesoftware/bioformats2raw/Converter.java
@@ -3252,7 +3252,8 @@ public class Converter implements Callable<Integer> {
    * @param args command line arguments
    */
   public static void main(String[] args) {
-    CommandLine.call(new Converter(), args);
+    int exitCode = new CommandLine(new Converter()).execute(args);
+    System.exit(exitCode);
   }
 
 }

